plugins/services/src/js/structs/Framework.js

```diff
@@ -1,15 +1,54 @@
 import { cleanServiceJSON } from "#SRC/js/utils/CleanJSONUtil";
 import { isSDKService } from "#PLUGINS/services/src/js/utils/ServiceUtil";
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
 
 import {
   ROUTE_ACCESS_PREFIX,
   FRAMEWORK_ID_VALID_CHARACTERS
 } from "../constants/FrameworkConstants";
 import FrameworkUtil from "../utils/FrameworkUtil";
+import ServiceStatus from "../constants/ServiceStatus";
 
 import Application from "./Application";
 import FrameworkSpec from "./FrameworkSpec";
 
+const frameworkStatusHelper = (displayName, key, priority) => ({
+  displayName,
+  key,
+  priority
+});
+
+// prettier-ignore
+/* eslint-disable no-multi-spaces */
+const FrameworkStatus = {
+  418: frameworkStatusHelper("Initializing",                    ServiceStatus.DEPLOYING,  1),
+  200: frameworkStatusHelper("Running",                         ServiceStatus.RUNNING,    1),
+  500: frameworkStatusHelper("Error Creating Service",          ServiceStatus.WARNING,    1),
+  204: frameworkStatusHelper("Deploying (Awaiting Resources)",  ServiceStatus.DEPLOYING,  2),
+  202: frameworkStatusHelper("Deploying",                       ServiceStatus.DEPLOYING,  2),
+  203: frameworkStatusHelper("Degraded (Awaiting Resources)",   ServiceStatus.RECOVERING, 4),
```

Comment:
Should this be `ServiceStatus.Warning` instead ? The design shows a red `CircleClose` icon for this status, not the `Yield` used for the `RECOVERING` status.